release-notes/5.16.0.md

```diff
@@ -0,0 +1,413 @@
+# CiviCRM 5.16.0
+
+In-progress, based upon commit 55086d4c32123382ce471afd2527469873120129
```

Comment:
When I check for late changes, I used to look for the last commit on the release notes branch prior to the "raw from script" commit.  I could then do something like https://github.com/civicrm/civicrm-core/compare/55086d4c32123382ce471afd2527469873120129...5.16 to compare it to the current state of the branch.  However, if we're merging the release notes as they're produced, there won't be that sort of easily-accessible record of when the script was run--and running the script earlier in the month means that there will be more late changes (even mid-month changes will be "late").

To resolve this, I'm putting this line in the release notes and then I'll remove it once the late changes are added.